sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 
 ### Bugs Fixed
 
+* Fixed a bug on deserialization of RunStepDeltaFileSearchToolCall [GitHub issue 48333](https://github.com/Azure/azure-sdk-for-net/issues/48333)
```

Comment:
Since this SDK includes things other than Agents, I think we should make it clear in the change log when an items is related to Agents. I can do that in a future PR, so don't spin a new build just for this change. But if you need to push another commit anyway, please update this line to make it clear this is for an Agent operation.

Reply:
Fixed.